Dockerfile

```diff
@@ -36,5 +50,23 @@ RUN wget --quiet https://chromedriver.storage.googleapis.com/89.0.4389.23/chrome
   && mv $PWD/chromedriver /usr/local/bin \
   && rm -f chromedriver_linux64.zip
 
+COPY --from=node-image /usr/local/bin/node /usr/local/bin/
+COPY --from=node-image /usr/local/lib/node_modules /usr/local/lib/node_modules
+RUN ln -s ../lib/node_modules/npm/bin/npm-cli.js /usr/local/bin/npm \
+    && ln -s ../lib/node_modules/npm/bin/npx-cli.js /usr/local/bin/npx
+
+RUN npm install -g \
+  jsdoc \
+  uglify-js \
+  prettier \
+  rollup \
+  rollup-plugin-terser
```

Comment:
I think we should also replace uglify-js with terser, which is more modern (supports ES6+), better maintained, and used in rollup. We can do it in a separate PR, just need to make sure it's installed here, which should be the case I think?

Reply:
See #1559.